src/game.cpp

```diff
@@ -344,7 +352,6 @@ void game::load_data_from_dir(const std::string &path)
 
 game::~game()
 {
-    DynamicDataLoader::get_instance().unload_data();
```

Comment:
That's exactly why I removed this line from the destructor.

> Things should clear themself and nothing outside.

That's what I meant by:

> but destroy themselves separately and independently.